src/AutoRest.CSharp.V3/Generation/Writers/CodeWriter.cs

```diff
@@ -118,7 +120,15 @@ public CodeWriter Append(FormattableString formattableString)
                     default:
                         if (isLiteral)
                         {
-                            Literal(argument);
+                            if (argument is EnumTypeValue enumTypeValue)
+                            {
+                                Constant constant = new Constant(argument, enumTypeValue.Declaration.Type);
+                                CodeWriterExtensions.WriteConstant(this, constant);
```

Comment:
Let's call WriteConstant from the spot that currently uses `:L` ande remove the code from here.